### ISSUE TITLE
tasmocompiler - Update language files

### ISSUE THREAD

**Issue by bovirus** [benzino77/tasmocompiler#134]
@drhideg
Could you please update hungarian language?
 
@davepee
Could you please update Czech language?

@benzino77
Could be possible contact translators or update the language file?
Many language file seems not updated.

**Comment by drhideg** [comment]
As I see it the Hungarian translation is already complete.

**Comment by bovirus** [comment]
@drhideg 

You are right. I change the term "update" with "check" that is more right.,

The current language status is

**- Complete/updated**

HU/IT

**- Not updated**

CZ/ES/PL

**Comment by benzino77** [comment]
PL is my native language, so it is always updated.
Authors of translations can be pointed by mentioning them - just like you did in your post. Unfortunately I don't have any other method to contact them.

**Comment by drhideg** [comment]
Could you add the Vetnam language to IT translation in #133? It was omitted when it was introduced.

**Comment by bovirus** [comment]
@drhideg 

Done.
Thanks for the info.

**Comment by ascillato** [comment]
Spanish Translation updated with https://github.com/benzino77/tasmocompiler/pull/157

**Comment by ascillato** [comment]
Fixed some typos on English with https://github.com/benzino77/tasmocompiler/pull/158

**Comment by ascillato** [comment]
Added Portuguese Translation with https://github.com/benzino77/tasmocompiler/pull/159

**Comment by bovirus** [comment]
No news for Czech language files?

**Comment by davepee** [comment]
Sorry for my long delay. Czech language seems to be OK. Only this is not translated ""Select Tasmota version and language." > "Vyberte verzi Tasmoty a jazyk".

**Comment by claymore666** [comment]
just for completion, updated german translations with [#257](https://github.com/benzino77/tasmocompiler/pull/257)
